client/state/sites/action-types.js

```diff
@@ -0,0 +1 @@
+export const RECEIVE_SITE = 'RECEIVE_SITE';
```

Comment:
I'd really like to stop using `keymirror` if we can. I don't see any real advantage over const refs.

One of the points of using it is that some optimizers can crush the keys (see https://github.com/facebook/react/issues/1639#issuecomment-45188026 ). This means that the action values we'll see in prod will be nonsense, which removes one of the great things about redux, the ability to replay the action queue and make sense of what happened.